src/arch/arm/isa/formats/fp.isa

```diff
@@ -2725,7 +2725,7 @@ let {{
             }
             break;
           case 0x9:
-            if ((opc3 & 0x1) == 0) {
+            if (bits(machInst, 6) == 1) {
                 if (single) {
```

Comment:
I noticed all the 16-bit operations are missing here and will decode as the double versions, probably also worth fixing but maybe in a follow-up commit?

Reply:
Yeah, I have a FEAT_FP16 implementation I plan to upstream sometime but I believe it is not covering AArch32